docker/README.md

```diff
@@ -0,0 +1,21 @@
+# Using the Dockerfiles
+
+To build the CPU version, run 
+```
+docker build --tag notebook --file cpu.Dockerfile .
+```
+and run with 
+```
+docker run --rm -it  --name notebook  --publish 8888:8888     notebook     jupyter-notebook --ip 0.0.0.0 --allow-root
```

Comment:
Could you please check if we can avoid using `--allow-root` and use instead `--user=$(id -u):$(id -g)`? (This flag runs the docker container with the current user and group.)

Reply:
Thanks! I made a new user in the container and gave it sudo access